.github/workflows/build.yml

```diff
@@ -408,4 +408,34 @@ jobs:
 
       - name: Run Tests
         run: npm test
-        working-directory: ./src/highlightjs
\ No newline at end of file
+        working-directory: ./src/highlightjs
```

Comment:
Btw, do we need to also update the official build? I can share instructions on how to test over teams.

Reply:
Not as of yet - these targets don't generate any actual artifacts, I'm just verifying they compile and the tests run.